src/hle/rt64_present_queue.cpp

```diff
@@ -406,6 +406,7 @@ namespace RT64 {
                 }
 
                 RenderCommandSemaphore *waitSemaphore = drawSemaphore.get();
+                ext.swapChain->wait();
```

Comment:
This was already added in a previous commit (check a few lines above). Should just be deleted.